### PR TITLE
Conda-build 3, update, fix up

### PR DESCRIPTION
**Pull request by pkgw**
This recipe has gotten a bit stale. This PR rolls up the conda-build 3 migration and the bump to the latest version, plus build fixes and improvements.

This may not build on macOS yet, but I need to push it here to be able to test it on my machine.

### CODE REVIEW

**Comment by pkgw** [comment]
@conda-forge/staged-recipes I'm not sure if the existing maintainer of this package is going to be very responsive; would appreciate a merge if it looks good and we don't hear anything very soon. This is blocking an update to gtk2 ([PR #1](https://github.com/conda-forge/gtk2-feedstock/pull/1)).

**Comment by scopatz** [comment]
Hi @notestaff - Are you still interested in maintaining this feedstock?  I'd be great if you are! If we don't hear back from you (one way or the other) in 5 days, I'll go ahead and merge this.

**Comment by jakirkham** [comment]
May want to look at PR ( https://github.com/conda-forge/gdk-pixbuf-feedstock/pull/5 ) to see if there is anything else needed for migration.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -61,3 +56,4 @@ about:
 extra:
   recipe-maintainers:
     - notestaff
+    - pkgw
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Thanks @pkgw!

**Comment by jakirkham** [comment]
Thanks all :)

**Pull request merged**